### PR TITLE
Merge Event and EventPage

### PR DESCRIPTION
**Pull request by patrys**
This makes event creation and management easier. The Event admin page was improved to show relevant data to both admins and organizers.

Redirects are provided from old EventPage admin URLs to the Event admin.

It also changes almost everything. Sorry 😭

### CODE REVIEW

File: core/admin.py [hunk 1]
```diff
@@ -37,15 +37,77 @@ def get_queryset(self, request):
...
+                ]}),
+                ('Event page', {'fields': [
+                    'page_title',
+                    'page_description',
+                    'page_main_color',
+                    'page_custom_css',
+                    'is_page_live'
+                ]})
+            ]
+        return [
```
Comment (olasitarska) [inline core/admin.py]: Organizers should be able to add / edit homepage photos themselves, but not the `is_on_homepage` parameter.

File: core/management/commands/copy_event.py [hunk 2]
```diff
@@ -64,46 +63,36 @@ def command():
     new_event.pk = None
     new_event.name = "{} #{}".format(name, number)
     new_event.date = date
+    new_event.is_page_live = False
+    new_event.page_custom_css = ''
```
Comment (olasitarska) [inline core/management/commands/copy_event.py]: We shouldn't clear the custom CSS when copying the event page?

File: templates/core/event.html [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
 {% block content %}
 
-    {% if messages or not page.is_live %}
+    {% if messages or not event.is_page_live %}
     <div class="container messages">
         <div class="row">
 
-            {% if not page.is_live%}
-            <div class="alert alert-info" role="alert">Hi there! You can see this page because you're logged in admin. When you want to make it public, go to <a href="/admin/core/eventpage/{{ page.pk }}/change/">your website section in the admin</a> and click on 'Is live'.</div>
+            {% if not event.is_page_live%}
```
Comment (olasitarska) [inline templates/core/event.html]: Missing whitespace ;))

**Comment by olasitarska** [comment]
Thanks to @asendecka for helping with this code review! <3

File: templates/core/event.html (hunk 3 above)
Comment (patrys) [inline templates/core/event.html]: This was already wrong but will fix it with the next commit.

File: core/management/commands/copy_event.py (hunk 2 above)
Comment (patrys) [inline core/management/commands/copy_event.py]: It seemed not to be preserved earlier. Will update.

**Comment by aniav** [comment]
Merging before our sprint starts 🎉

**Pull request merged**